src/hotspot/share/runtime/os.hpp

```diff
@@ -81,6 +81,11 @@ enum ThreadPriority {        // JLS 20.20.1-3
   CriticalPriority = 11      // Critical thread priority
 };
 
+enum WXMode {
+  WXWrite,
+  WXExec
+};
+
```

Comment:
Why is this defined for all arches rather than just for MACOS_AARCH64?

Is it ever meant to be used outside of a MACOS_AARCH64_ONLY macro or #ifdef